argocd/overlays/moc-infra/applications/envs/moc/smaug/chris/chris.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: chris
+spec:
+  destination:
+    name: smaug
+    namespace: chris-dev
+  source:
```

Comment:
looks like you want to deploy a helm chart, for this the declarative options are different, see here: https://argo-cd.readthedocs.io/en/stable/user-guide/helm/

Reply:
@HumairAK , it would be a kustomize in this case

Reply:
@HumairAK this is fixed